frontend/src/features/admin-form/create/logic/components/EmptyLogic.tsx

```diff
@@ -24,27 +37,31 @@ export const EmptyLogic = (): JSX.Element => {
       color="secondary.500"
       pt={{ base: '0.5rem', md: '2.75rem' }}
     >
-      <Text textStyle="h2" as="h2">
-        Start creating logic for your form
-      </Text>
-      <Text textStyle="body-1" mt="1rem">
-        Show or hide fields depending on user input, or disable form submission
-        for invalid answers.{' '}
-        <Link isExternal href={GUIDE_FORM_LOGIC}>
-          Learn to work with logic
-        </Link>
-      </Text>
-      <Button
-        my="2.5rem"
-        leftIcon={<BiPlus fontSize="1.5rem" />}
-        onClick={setToCreating}
-      >
-        Add logic
-      </Button>
-      <LogicSvgr maxW="292px" />
-      <Divider my="2.5rem" />
+      <Skeleton isLoaded={isLoaded}>
```

Comment:
lmao I see this component has both the empty and loading states. I suppose implementing the loading state is a loooot easier if you can just wrap the empty state. what dyou think about renaming the component to `EmptyOrLoadingLogic`? just because we normally think of "empty" and "loading" as separate categories, rather than "loading" being a subset of "empty"

Reply:
The reason for doing it this way is because the design on Figma was modelled after the empty state, so it made sense to simply wrap the empty state. It's also possible to just create a new skeleton component though, depends on how we want the skeleton to be shown! We can discuss this a bit more at sync.

Reply:
discussed at sync, loading screen has been redesigned!